components/layout_2020/dom_traversal.rs

```diff
@@ -228,7 +228,7 @@ fn traverse_pseudo_element_contents<'dom, Node>(
                         .stylist
                         .style_for_anonymous::<Node::ConcreteElement>(
                             &context.shared_context().guards,
-                            &PseudoElement::ServoText,
+                            &PseudoElement::ServoAnonymousBlock,
```

Comment:
Actually this is not a block, there is even an assert for `display: inline`.
Maybe it should be renamed to `ServoAnonymousBox`?

Reply:
Okay. I've gone ahead and renamed this `PseudoElement` variant.